### PR TITLE
fix(drm): add tick_get_cb

### LINKED ISSUE #6266: Linux DRM display mode locks up by default due to missing lv_tick_set_cb
### LVGL version

v9.1.0-182

### What happened?

When testing different Linux display drivers I noticed that the legacy framebuffer one was working, but the DRM/KMS got stuck and didn't render anything besides the initial screen.

Issue seems to be the `lv_tick_set_cb` call being present on the legacy framebuffer option, but not on the DRM/KMS one.
Issue is not present if that call is done by the user.

For consistency both should have the tick call, or maybe neither..?

### How to reproduce?

_No response_

### PR DESCRIPTION
**Pull request by kisvegabor**
fixes #6266

### Description of the feature or fix

A clear and concise description of what the bug or new feature is.

### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` ([astyle](http://astyle.sourceforge.net/install.html) version [v3.4.12](https://github.com/szepeviktor/astyle/releases/tag/v3.4.12) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html).
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### CODE REVIEW

File: src/drivers/display/drm/lv_linux_drm.c [hunk 1]
```diff
@@ -109,6 +112,13 @@ static void drm_flush(lv_display_t * disp, const lv_area_t * area, uint8_t * px_
 
 lv_display_t * lv_linux_drm_create(void)
 {
+    static bool inited = false;
+
+    if(!inited) {
```
Comment (FASTSHIFT) [inline src/drivers/display/drm/lv_linux_drm.c]: No judgment is required, and it does not matter if the settings are repeated.

File: src/drivers/display/drm/lv_linux_drm.c [hunk 2]
```diff
@@ -836,4 +846,13 @@ static void drm_flush(lv_display_t * disp, const lv_area_t * area, uint8_t * px_
...
 }
 
+static uint32_t tick_get_cb(void)
+{
+    struct timeval tv_now;
+    gettimeofday(&tv_now, NULL);
+    uint64_t time_ms;
+    time_ms = (tv_now.tv_sec * 1000000 + tv_now.tv_usec) / 1000;
+    return time_ms;
+}
```
Comment (liamHowatt) [inline src/drivers/display/drm/lv_linux_drm.c]: @kisvegabor please see this: #6296 :slightly_smiling_face: 

`clock_gettime(CLOCK_MONOTONIC, &period);` gets the time from a clock whose value doesn't jump when the system time changes. Can we use that here?

File: src/drivers/display/drm/lv_linux_drm.c (hunk 1 above)
Comment (kisvegabor) [reply src/drivers/display/drm/lv_linux_drm.c]: Fixed

File: src/drivers/display/drm/lv_linux_drm.c (hunk 2 above)
Comment (kisvegabor) [reply src/drivers/display/drm/lv_linux_drm.c]: Updated

**Review by kisvegabor — COMMENTED** [review]
Thanks, both are fixed!

**Review by liamHowatt — APPROVED** [review]

**Review by FASTSHIFT — APPROVED** [review]

**Pull request merged**